packages/typography/src/useUpdatedBaseFontSize.ts

```diff
@@ -0,0 +1,13 @@
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { BaseFontSize } from '@leafygreen-ui/tokens';
+
+/**
+ * A wrapper around the hook useBaseFontSize that returns
+ * a value that is consistent with the updated base font sizes.
+ *
+ * Prefer using {@link useBaseFontSize} when LeafygreenProvider is updated to support the new values.
+ */
+export const useUpdatedBaseFontSize = (): BaseFontSize => {
+  const baseFontSize = useBaseFontSize();
+  return baseFontSize === 16 ? baseFontSize : 13;
+};
```

Comment:
We'll need to export this from the package's `index.ts`.

Reply:
If it's only public-ish, wouldn't importing it from `/dist/useUpdatedBaseFontSize` be fine?

Reply:
Also, I went back & forth on adding this in `typography` vs in `leafygreen-provider` (where the original hook is). Went with typography since I'd need to add `tokens` as  a dependency to the provider, which is a bigger change than I think this warrants. Thought?

Reply:
I'd rather us not import that way. It's a bit more brittle if we make build or package structure changes (though I'm aware we do this with Icon).

I don't mind it living in this package because it's temporary.